weblate/formats/convert.py

```diff
@@ -240,12 +240,12 @@ def convertfile(storefile, template_store):
 
     def save_content(self, handle):
         """Store content to file."""
-        convertor = po2html()
+        converter = po2html()
```

Comment:
@nijel, I am afraid that this could break the code?

Reply:
Tests pass, so it should be okay :-).